seed/seedtest/seedtest.go

```diff
@@ -337,3 +339,30 @@ func (s *TestingSeed20) MakeSeed(c *C, label, brandID, modelID string, modelHead
 
 	return model
 }
+
+func ValidateSeed(c *C, root, label string, trusted []asserts.Assertion) seed.Seed {
```

Comment:
I don't know the bigger picture of the upcoming changes, but  this helper was a private helper for some tests, now it is going to be exported for reuse but it seems it has very concrete assumptions about seed, e.g. is the number of expected essential snaps always 4 with core20?

Should it be renamed if it assumes core20 seeds?

Reply:
Hmm I should probably rename it to ValidateCore20Seed then

Reply:
It seems like it's kinda general, except for assuming snapd and 4 essential snaps; maybe pass them as args to make it more reusable?

Reply:
I think just renaming the helper for now is fine, we can make it more generic when it needs to be more generic I suppose

Reply:
I've tweaked the parameters to make it work with core20/core18/core16 seeds and tweaked the seedwriter tests to call the helper